src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -24,7 +24,7 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\pkg\System.IO.FileSystem.pkgproj">
+    <ProjectReference Include="..\src\System.IO.FileSystem.csproj">
```

Comment:
We shouldn't be directly referencing csproj files from the projects as that means it will not get the correct binaries for the different targets.